mclocations/minecraft.py

```diff
@@ -0,0 +1,89 @@
+import re
+
+from mcrcon import MCRcon
+
+from mclocations.models import Player, Position
+
+
+class MinecraftServer:
+    def __init__(self, host, password, port=25575):
+        self.mcr = MCRcon(host, password, port)
+        self.mcr.connect()
+
+    def disconnect(self):
+        self.mcr.disconnect()
+
+    def get_players(self):
+        players = []
+        player_names = MinecraftServer._parse_list(self.mcr.command('/list'))

```

Comment:
Just as an FYI, there is a way to do this and get all of the player names and location data in one command. It still needs obnoxiously parsed, but one command seems better than _number_of_players*2+1_ calls

```
execute as @p run data get entity @s Pos
```
```
execute as @p run data get entity @s Dimension
```

It will return a string of:

> PlayerName has the following entity data: [565.6831868873832d, 16.0d, -1096.8317977225513d]
AnotherPlayerName has the following entity data: [565.6831868873832d, 16.0d, -1096.8317977225513d]

There are new lines, but I think they may be stripped in the rcon parser.